components/automate-chef-io/content/docs/saml.md

```diff
@@ -49,23 +49,26 @@ Users who sign in via SAML will have a session time of 24 hours before needing t
 - Ping
 - Tivoli Federated Identity Manager
 
-In the following section, you can find further details about specific supported identity management systems.
-
 ### Azure AD
 
-For using Chef Automate's SAML integration with Azure AD, please note that **signing key rotation** is
-not supported to be done automatically. The signing certificate used with Azure for SAML sign-ins
-has to be **managed manually**.
+Using Azure AD as an SAML IdP requires specific configuration for both Azure AD and Chef Automate.
+
+{{< info >}}
+The signing certificate used for Chef Automate's SAML integration with Azure AD requires manual management.
+Signing key rotation is not done automatically.
+{{< /info >}}
 
-After setting up your application as a _"non-gallery application"_, configure its SAML sign-on method.
-For both _Identifier (Entity ID)_ and _Reply URL (Assertion Consumer Service URL)_, enter
-`https://{{< example_fqdn "automate" >}}/dex/callback`.
+First, set up your configuration within Azure AD.
+Set up your application as a _"non-gallery application"_, and then configure its SAML log-in method.
```

Comment:
I believe we've been trying to standardize on "sign-in" instead of "log-in" -- am I on the wrong track here?